scripts/format.sh

```
@@ -0,0 +1,4 @@
+#!/bin/zsh
```

Shellcheck (0.11.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)